README.md

```diff
@@ -17,14 +17,16 @@ solution. You can find the sample code for the Android client code in the
 - Sign up for [Firebase](https://firebase.google.com/) and create a new project
   in the [Firebase console](https://console.firebase.google.com/).
 - Install the following tools in your development environment:
-  - [Python 2.7](https://www.python.org/downloads/)
```

Comment:
Happy to remove stuff that is not needed, but could you explain your rationale?

Reply:
Python 2.7 is not a direct dependency for this project. It's required by the Cloud SDK, which supports both Python 2 and 3.
https://cloud.google.com/sdk/install